services/src/main/java/org/jboss/windup/web/services/messaging/WindupExecutionTask.java

```diff
@@ -55,6 +55,10 @@ public void init(WindupExecution execution, ApplicationGroup group)
     @Override
     public void run()
     {
+        if (this.execution.getState() == ExecutionState.CANCELLED) {
```

Comment:
It seems to me that this is unlikely to work. If the message to execute has already been serialized onto the wire via JMS, how would the state have been changed to "CANCELLED"?